README.md

```diff
@@ -1 +1,36 @@
 # ui-claim-tracker
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
```

Comment:
Let's keep the README updated as we go along! so maybe something like

git clone this repo
brew install yarn
yarn install
yarn dev

and delete everything on line 30 and later